src/components/vm/disks/diskEdit.jsx

```diff
@@ -126,7 +126,7 @@
                        name="access"
                        value="writable"
                        isChecked={dialogValues.access == "writable" }
-                       onChange={event => {
+                       onChange={(event, _) => {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1088-20230629-150027-4152f662-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)